ocs_ci/ocs/constants.py

```diff
@@ -153,6 +154,8 @@
 DEFAULT_NOOBAA_BACKINGSTORE = "noobaa-default-backing-store"
 RIPSAW_NAMESPACE = "my-ripsaw"
 RIPSAW_CRD = "resources/crds/ripsaw_v1alpha1_ripsaw_crd.yaml"
+OCP_QE_DEVICEPATH_REPO = "https://github.com/anubhav-here/device-by-id-ocp.git"
```

Comment:
I imagine this is your fork. Is there a reason we are not going with the original repo?

Reply:
yes, I did make some changes in it after the fork. Also, he is using that repo for some other use case in other repositories. Hence, I thought it was safe to use my fork.